src/index.ts

```diff
@@ -43,6 +43,7 @@ class MagicNixCacheAction extends DetSysAction {
       fetchStyle: "gh-env-style",
       idsProjectName: "magic-nix-cache-closure",
       requireNix: "warn",
+      diagnosticsSuffix: "perf",
```

Comment:
I noticed the update-flake-lock doesn't have this set. And `perf`/`telemetry`, not sure the name scheming? Of course, maybe this is just temporary for an initial run through CI checks?

Reply:
The projects don't have a consistent name for them for historical reasons. This suffix matches what was in the action.yml.